apps/dc_tools/odc/apps/dc_tools/add_update_products.py

```diff
@@ -121,7 +121,7 @@ def cli(csv_path: str, update_if_exists: bool):
     # TODO: Add in some QA/QC checks
     added, updated, failed = add_update_products(dc, csv_path, update_if_exists)
 
-    logging.info(f"Added: {added}, Updated: {updated} and Failed: {failed}")
+    print(f"Added: {added}, Updated: {updated} and Failed: {failed}")
```

Comment:
Personal preference is is logging to print, and debug using log levels in environment. I will try to add that as an enhancement to the CLI tools. https://docs.python.org/3/howto/logging.html

Reply:
I changed this so that it was consistent with some of the other tools, and with testing using `result.output` with `CliRunner`.

Reply:
Logging with info here might mean that this is hidden.